scripts/pylint_extensions.py

```diff
@@ -134,6 +139,33 @@ def visit_call(self, node):
                             func_name))
                     num_positional_args_unused -= 1
 
+            #Executed if name is not having any default value in the function
+            #definition
+            else:
+                if name is None:
+                    display_name = '<tuple>'
+                else:
+                    display_name = repr(name)
+
+                if name in keyword_args:
+                    # This try/except block tries to get the function
+                    # name. Since each node may differ, multiple
+                    # blocks have been used.
+                    try:
+                        func_name = node.func.attrname
+                    except AttributeError:
+                        try:
+                            func_name = node.func.name
+                        except AttributeError:
+                            func_name = node.fun
```

Comment:
What is `node.fun`? Please test all your changes before pushing the code.

Reply:
My bad. Noted.